qemu/tests/hotplug_mem_negative.py

```diff
@@ -60,6 +63,15 @@ def check_msg(keywords, msg):
         if not re.search(r"%s" % keywords, msg):
             test.fail("No valid keywords were found in the qemu prompt message")
 
+    qemu_path = utils_misc.get_qemu_binary(params)
+    qemu_version = env_process._get_qemu_version(qemu_path)
+    version_pattern = r'%s-(\d+\.\d+\.\d-\d+)' % os.path.basename(qemu_path)
+    host_qemu = re.findall(version_pattern, qemu_version)[0]
+
+    if host_qemu in VersionInterval('[8.2.0-1,8.2.0-4]'):
```

Comment:
Isn't this version caused by a product bug? So if it fails I understand it is expected, 
@yanan-fu  what do you think?

Reply:
Yes, actually the case failed due to the bug you reported :grinning: 
IMO if we can avoid a failed case, let's do it

Reply:
If i understand correctly,   QEMU 8.2 rebase involve a regression issue and be fixed now ?
I have same opinion with @zhenyzha ,  we do not need this patch any more, test fail is the expected result.

Reply:
> If i understand correctly, QEMU 8.2 rebase involve a regression issue and be fixed now ?

Yes, that's it, so if no actions are needed then I will close the PR